bundles/org.openhab.binding.cloudrain/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="cloudrain"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<!-- ############################### Irrigation Zone ############################### -->
+
+	<thing-type id="zone">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="account"/>
+		</supported-bridge-type-refs>
+
+		<label>Irrigation Zone</label>
+		<description>Represents an irrigation zone as defined in Cloudrain</description>
+
+		<channel-groups>
+			<channel-group id="irrigation" typeId="irrigation"/>
+			<channel-group id="command" typeId="command"/>
+		</channel-groups>
+		<properties>
+			<property name="uid"/>
+			<property name="zoneId"/>
+			<property name="zoneName"/>
+			<property name="controllerId"/>
+			<property name="controllerName"/>
+		</properties>
+	</thing-type>
+
+	<!-- ############################### channels and groups ############################### -->
+
+	<channel-group-type id="irrigation">
+		<label>Irrigation Properties</label>
+		<description>Properties of an irrigation in a zone</description>
+		<channels>
+			<channel id="state" typeId="state">
+				<label>Irrigation Status</label>
+			</channel>
+			<channel id="startTime" typeId="timestamp">
+				<label>Start Timestamp</label>
+			</channel>
+			<channel id="plannedEndTime" typeId="timestamp">
+				<label>Planned End Timestamp</label>
+			</channel>
+			<channel id="remainingSeconds" typeId="durationSec">
+				<label>Remaining Seconds</label>
+			</channel>
+			<channel id="duration" typeId="durationSec">
+				<label>Duration</label>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="command">
+		<label>Irrigation Commands</label>
+		<description>Commands to control an irrigation in a zone</description>
+		<channels>
+			<channel id="commandDuration" typeId="irrigationCommandDuration">
+				<label>Irrigation Duration</label>
+				<description>The duration in seconds to be used for the next command.</description>
+			</channel>
+			<channel id="startIrrigation" typeId="cloudrainCommand">
+				<label>Start irrigation</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Fixed.